drivers/node/ssh/ssh.go

```diff
@@ -17,9 +17,9 @@ const (
 	// DriverName is the name of the ssh driver
 	DriverName = "ssh"
 	// DefaultUsername is the default username used for ssh operations
-	DefaultUsername = "torpedo"
-	// DefaultPassword is the default username used for ssh operations
-	DefaultPassword = "t0rped0"
+	DefaultUsername = "root"
+	// DefaultPassword is the default password used for ssh operations
+	DefaultPassword = ""
```

Comment:
why even have the default password field if we are not setting it up

Reply:
The other option was to set it down in this file as an empty string or add it as a constant up here. Just future proofing.